app/views/layouts/application.html.haml

```diff
@@ -24,10 +24,7 @@
       ManageIQ.browser = "#{j browser_info(:name_ui)}";
       ManageIQ.controller = "#{j controller_name}";
       ManageIQ.timezone = '#{j ::ActiveSupport::TimeZone::MAPPING[session[:user_tz]]}';
-
-    - if ::Settings.server.asynchronous_notifications
-      :javascript
-        miqInitNotifications();
+      ManageIQ.asynchronous_notifications = #{::Settings.server.asynchronous_notifications}
```

Comment:
missing semicolon,
and also add `asynchronous_notifications` to `miq_global.js` (and I guess delete `.notifications`?)

Reply:
Also, what are the allowed values here?

This will only work for strings (or `nil` which will get converted to "").

Reply:
There should be an explicit type conversion here..
wrap in quotes if a string,
to_json if object,
!! if bool

Reply:
I think it's a rails bug that `j()` doesn't work on a boolean automagically :confused: